pilot/pkg/networking/v1alpha3/config.go

```diff
@@ -190,6 +191,9 @@ func buildHTTPConnectionManager(opts buildHTTPListenerOpts) *http_conn.HttpConne
 		}
 	*/
 	refresh := time.Duration(opts.mesh.RdsRefreshDelay.Seconds) * time.Second
+	if refresh == 0 {
+		refresh = 5 * time.Second
```

Comment:
define this as a constant.

Reply:
Added comment - it's temp code for tests, until we get to v2 RDS, not worth adding a const (to remove later). Besides: "5" is constant :-)